test/e2e-helm/e2e_helm_olm_test.go

```diff
@@ -26,18 +26,14 @@ var _ = Describe("Integrating Helm Projects with OLM", func() {
 		const operatorVersion = "0.0.1"
 
 		It("should generate and run a valid OLM bundle and packagemanifests", func() {
-			By("building the operator bundle image")
-			err := tc.Make("bundle-build", "BUNDLE_IMG="+tc.BundleImageName)
-			Expect(err).NotTo(HaveOccurred())
-
 			if tc.IsRunningOnKind() {
 				By("loading the bundle image into Kind cluster")
-				err = tc.LoadImageToKindClusterWithName(tc.BundleImageName)
+				err := tc.LoadImageToKindClusterWithName(tc.BundleImageName)
 				Expect(err).NotTo(HaveOccurred())
 			}
 
 			By("adding the 'packagemanifests' rule to the Makefile")
-			err = tc.AddPackagemanifestsTarget()
+			err := tc.AddPackagemanifestsTarget()
```

Comment:
> I'm not a fan of separating OLM integration setup code into BeforeEach and test specs for the sake of encapsulation/refactoring. Can you call IntegrateProjectWithOLM in the test specs? It may be a bit of duplicated code but it is clearer.

What the `IntegrateProjectWithOLM` should does?